src/components/DiscriminationFreeSpaces.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <section>
+    <ul class="discrimination-free-spaces__list">
+      <li class="discrimination-free-spaces__item" v-for="(entity, index) in nodeQuery.entities">
+        <a class="discrimination-free-spaces__link" :href='nodeQuery.entities[index].fieldEnlace.url.path'>
+          <img class="discrimination-free-spaces__img" :src="nodeQuery.entities[index].fieldImagen.derivative.url" alt="">
+        </a>
+      </li>
+    </ul>
+  </section>
+</template>
+
+<script>
+import gql from 'graphql-tag';
+
+const query = gql`query {
+  nodeQuery(limit: 6,
+    sort: [
+      {
+        field: "created",
+        direction: DESC
+      }
+    ],
+    filter: {
+      conditions: [
+        {
+          operator: EQUAL, field: "type",
+          value: ["espacio_libre_de_discriminacion"]
+        }
+      ]
+    }) {
+    entities {
+      ... on NodeEspacioLibreDeDiscriminacion {
+          fieldImagen {
+        	  derivative (style: medium) {
+          	  url
+          }
+      	}
+        fieldEnlace {
+          url {
+            path
+            alias
+          }
+        }
+      }
+    }
+  }
+}`;
+
+export default {
+  name: 'DiscriminationFreeSpaces',
+  apollo: {
+    nodeQuery() {
+      return {
+        query: query,
+        update(data) {
+          if (data.hasOwnProperty('nodeQuery') &&
+            data.nodeQuery.hasOwnProperty('entities') &&
+            data.nodeQuery.entities instanceof Array &&
+            data.nodeQuery.entities.length) {
+            const entities = data.nodeQuery.entities;
+            return {
+              entities: entities,
```

Comment:
`update()` function is not necessary, because you're returning the data as it came. If you're not going to manipulate the data, you can omit this function.